### PR TITLE
Add a how_tos file to show an example of settle a message using negative acknowledgement

### PR DESCRIPTION
**Pull request by khoigithub**

### CODE REVIEW

File: howtos/pubsub/how_to_consume_persistent_message_with_negative_acknowledgement.py
Comment (gszol) [inline howtos/pubsub/how_to_consume_persistent_message_with_negative_acknowledgement.py]: I only tried a quick case sensitive search in my browser, but this seems unused.

File: howtos/pubsub/how_to_consume_persistent_message_with_negative_acknowledgement.py
Comment (gszol) [inline howtos/pubsub/how_to_consume_persistent_message_with_negative_acknowledgement.py:88]: Maybe print the stats at the end.

**Review by gszol — APPROVED** [review]
Looks good, thank you!

File: howtos/pubsub/how_to_consume_persistent_message_with_negative_acknowledgement.py
Comment (khoigithub) [reply howtos/pubsub/how_to_consume_persistent_message_with_negative_acknowledgement.py:88]: Ah ok sounds good, I will print the stats down below

**Comment by TrentDaniel** [comment]
@gszol, what's the state of this PR? How much work is left for this sample?

**Comment by gszol** [comment]
> @gszol, what's the state of this PR? How much work is left for this sample?

It's good to go, it just has to wait till the release to go public.

**Comment by TamimiGitHub** [comment]
I'm assuming this will also require a bump in the solace package version in the requirements.txt file correct?

**Review by gszol — COMMENTED** [review]
It's OK. More in line with the established "sampler" style than I ever could go.

This line does nothing though, I even tried running it after deleting it in case it's something magical:
X = TypeVar('X')

I don't have write access to the source repo, so I'm not sure what's the simplest way to go about deleting it.

**Review by gszol — APPROVED** [review]
Awesome, thank you!

**Review by TamimiGitHub — APPROVED** [review]
Ran it and tested it. LGTM!

**Pull request merged**
